timestamp.go

```diff
@@ -154,8 +175,12 @@ type Timestamp struct {
 	HashedMessage []byte
 
 	Time         time.Time
-	Accuracy     time.Duration
+	Accuracy     Accuracy
```

Comment:
Please keep the accuracy as `time.Duration` this can be calculated as following: https://play.golang.org/p/9HdBIdYJ4Pd

Reply:
Done.